airbyte-integrations/bases/base-normalization/requirements.txt

```diff
@@ -1,3 +1,2 @@
 -e ../airbyte-protocol
 -e .
-dbt==0.18.1
```

Comment:
i think someone put a non-local dependency here, because setup.py wasn't doing the installation.

Reply:
Yes, I added that there at some point to make some task work somehow... Not sure if you remove it, it's going to do well or not... 

The way Airbyte is building python packages is really weird and i don't think it's following proper python recommendations, for example I don't understand why we do some things in setup.py and are using requirements.txt files in this way... (see [here](https://packaging.python.org/discussions/install-requires-vs-requirements/)

Then lately i also run in such build failures that could be related:
![Screenshot 2020-11-30 at 14 21 26](https://user-images.githubusercontent.com/7718175/100615595-1802cb00-3318-11eb-8316-2d0a1c9b33fa.png)